tests/Soak.rb

```diff
@@ -6,7 +6,7 @@
   perf_teardown
 end
 
-# Execute 60 agent runs to warm up the JIT before starting our monitoring.
+# Execute agent runs to warm up the JIT before starting our monitoring.
```

Comment:
Since you are adding specifics... the target is 300 hits on the endpoint... currently 60 of our agent runs gets that because our recording hits 5 times.  That 5 times may change, so if being specific say "Execute 60 agent runs to hit the endpoints 300 times to warm up..."  Or something like that.

Reply:
Updated to "Execute 60 agent runs (hitting the endpoints 300 times) to warm up the JIT before starting our monitoring."